test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -90,11 +90,11 @@ private void assertCommandBehavior(String inputCommand,
         //Confirm the state of data is as expected
         assertEquals(expectedAddressBook, addressBook);
         assertEquals(lastShownList, logic.getLastShownList());
-        assertEquals(addressBook, saveFile.load());
+//        assertEquals(addressBook, saveFile.load());
```

Comment:
Why did you remove this line of code? Doesn't this remove an aspect of the test?
Try to edit the test cases such that they still retain their original intention, rather than remove the test.